playbooks/lvm.yml

```diff
@@ -0,0 +1,43 @@
+---
+- name: lvm
+  hosts: all
+  tasks:
+
+    - name: ensure values have been provided
+      fail:
+        msg: "Please provide necessary values."
+      when:
+        lvm.volume_group is not defined or
+        lvm.physical_volumes is not defined or
+        lvm.logical_volume is not defined or
+        lvm.mount_point is not defined
+
+    - name: install dependencies
+      apt:
+        pkg: lvm2
+        state: present
+
+    - name: create volume group
+      lvg:
+        vg: "{{ lvm.volume_group }}"
+        pvs: "{{ lvm.physical_volumes }}"
+        pesize: "{{ lvm.physical_extent_size }}"
+
+    - name: create logical volume
+      lvol:
+        vg: "{{ lvm.volume_group }}"
+        lv: "{{ lvm.logical_volume }}"
+        size: "{{ lvm.size }}"
```

Comment:
Should this maybe loop over a hash of LVs to create, so a system can have more than one easily defined in the vars?